notebooks/test-mixture-fitting.ipynb

```diff
@@ -10,7 +10,7 @@
     "import seaborn\n",
     "\n",
     "from ergo import Logistic, LogisticMixture\n",
-    "from ergo.distributions.conditions import HistogramCondition, IntervalCondition, PercentileCondition\n",
+    "from ergo.distributions.conditions import IntervalCondition, PercentileCondition\n",
```

Comment:
Did you look over the notebooks? Are any of them broken as a result of this PR?

Reply:
I just spent some time looking at and trying to run the notebooks on master, and I think cleaning them up is outside the scope of this PR. None of the 5 notebooks I tried to run in Colab succeeded, though some of the errors seem mostly minor. One example of the notebooks not being kept up to date is the line you commented on importing `PercentileCondition`, which doesn't currently exist.